### DISCUSSION TITLE
[网站bug] 网站的 element UI 的 css 文件被注释了

### DISCUSSION THREAD

**Discussion by lanjingio** [Gamuxorg/bbs#331] [游戏或网站运行反馈]
非常影响访问

**Comment by shouhuanxiaoji** [comment]
是否可截图说明一下？

**Comment by lanjingio** [comment]
![截图 2022-09-01 18-13-51](https://user-images.githubusercontent.com/48174168/187890639-67e74392-30e6-4f17-bf9d-e8c33cc22a1b.png)
在没有插件的环境也是如此

**Comment by hyh19962008** [comment]
控制台有报错吗？你是通过哪个地址访问的？
注释掉的样式表实际上用的是下一行

Best

________________________________

**Comment by lanjingio** [comment]
访问地址：https://www.linuxgame.cn/gamelist

请求网址: https://cdn.bootcdn.net/ajax/libs/element-ui/2.15.3/theme-chalk/index.min.css
请求方法: GET
状态代码: 521 
远程地址: 120.232.175.190:6228
引荐来源网址政策: strict-origin-when-cross-origin

请求标头:
:authority: cdn.bootcdn.net
:method: GET
:path: /ajax/libs/element-ui/2.15.3/theme-chalk/index.min.css
:scheme: https
accept: text/css,*/*;q=0.1
accept-encoding: gzip, deflate, br
accept-language: zh-CN,zh;q=0.9
cache-control: no-cache
dnt: 1
pragma: no-cache
referer: https://www.linuxgame.cn/
sec-ch-ua: "Chromium";v="104", " Not A;Brand";v="99", "Google Chrome";v="104"
sec-ch-ua-mobile: ?0
sec-ch-ua-platform: "Linux"
sec-fetch-dest: style
sec-fetch-mode: no-cors
sec-fetch-site: cross-site
user-agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/104.0.0.0 Safari/537.36

**Comment by shouhuanxiaoji** [comment]
查了一下，bootcdn和baomitu这俩前端镜像站都挂了。。。尽快修复

**Comment by shouhuanxiaoji** [comment]
已修复
